tools/sync_test_server/ros-testing-server.js

```diff
@@ -64,10 +65,16 @@ function stopRealmObjectServer() {
     if (syncServerChildProcess) {
         syncServerChildProcess.kill();
         syncServerChildProcess = null;
+        exec('rm -r ' + 'realm-object-server', function (err, stdout, stderr) {
```

Comment:
We need to do this according to @mrackwitz . I agree It does feel rather odd though since `--path` is defined as `--root ROOT_PATH`.